tests/integration/s3/test_s3.py

```diff
@@ -3887,6 +3887,89 @@ def test_s3_sse_default_kms_key(
         response = aws_client.s3.get_object(Bucket=bucket_1, Key=key_name)
         snapshot.match("get-obj-default-kms-s3-key-from-bucket", response)
 
+    @pytest.mark.aws_validated
+    @pytest.mark.skip_snapshot_verify(paths=["$..ServerSideEncryption"])
```

Comment:
Why are we skipping to verify the path `ServerSideEncryption` ?

Reply:
Because AWS now has server side encryption by default on all newly created buckets: https://docs.aws.amazon.com/AmazonS3/latest/userguide/default-encryption-faq.html

This behaviour is not yet in moto or LocalStack, so we need to skip the return value for now. All newly created tests now have to have this skip until we fix it 😞